src/en/get-involved.md

```diff
@@ -15,31 +15,37 @@ templateEngineOverride: njk,md
 
 A design system is never complete. We're committed to building with the people who use our products.
 
-Right now, we're testing GC Design System in alpha, the first usable phase of a product. We're iterating based on your needs and those of the people you serve.
-
-## Find out what we're up to
-
-We're sharing what we're working on and what we'll work on next. To get updates on GC Design System, sign up for our mailing list through our [Contact form]({{ links.contact }}).
+Right now, we're testing GC Design System in alpha, the first usable phase of a product. We’re iterating based on common needs across the GC and those of the people we all serve.
+
+## Find out about GC Design System
+
+<gcds-card
+  card-title="Subscribe to newsletter"
+  href="#https://design-system.alpha.canada.ca/en/contact/"
+  badge="badge"
+  description="Subscribe to our newsletter to get GC Design System updates, release communications, and special events. "
+>
+<gcds-card
+  card-title="Attend a demo"
+  href="#https://design-system.alpha.canada.ca/en/register-for-a-demo/" 
+  badge="badge"
+  description="Get an intro to prototyping and developing web experiences using the design system, followed by a Q&A. "
+>
```

Comment:
This change will make the cards be side by side, fixes the render of the card, removes the badge and fixes the links.

```suggestion
<gcds-grid class="mb-400" columns-tablet="1fr 1fr">
  <gcds-card
    card-title="Subscribe to newsletter"
    href="{{ links.contact }}"
    description="Subscribe to our newsletter to get GC Design System updates, release communications, and special events."
  ></gcds-card>
  <gcds-card
    card-title="Attend a demo"
    href="{{ links.registerDemo }}"
    description="Get an intro to prototyping and developing web experiences using the design system, followed by a Q&A."
  ></gcds-card>
</gcds-grid>
```